src/main/java/duke/operation/TaskList.java

```diff
@@ -0,0 +1,214 @@
+package duke.operation;
+
+import duke.Parser.Parser;
+import duke.exception.DukeException;
+
+import java.util.ArrayList;
+
+/**
+ * This is the TaskList class to contain task list and handle operations.
+ */
+public class TaskList {
+	protected ArrayList<Task> taskList;
+
+	public TaskList() {
+		this.taskList = new ArrayList<>();
+	}
+
+	public TaskList(ArrayList<Task> list) {
+		this.taskList = list;
+	}
+
+	public ArrayList<Task> getTaskList() {
+		return taskList;
+	}
+
+	/**
+	 * Adds tasks to the storeroom.
+	 *
+	 * @param command   command name
+	 * @param inputLine input string
+	 * @throws DukeException
+	 */
+	public void addTask(Command command, String inputLine) throws DukeException {
```

Comment:
Perhaps you can extract out certain sections of the code such as the constants for printAddTask to a variable outside the method since the method is too long and according to the coding standards here methods should not be longer than 30 LOC [here](https://nus-cs2103-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)